### PR TITLE
PS-7678 Document deprecation of TokuDB on "Changed in Percona Server …

### PR DESCRIPTION
**Pull request by patrickbirch**
…8.0" (8.0)

 Changes to be committed:
	modified:   source/changed_in_version.rst

### CODE REVIEW

**Review by LenzGr — APPROVED** [review]
LGTM, thanks!

File: doc/source/changed_in_version.rst [hunk 1]
```diff
@@ -149,3 +149,7 @@ Removed System variables
   which was an alias to the ``kill_idle_trx`` system variable
 - The `max_binlog_files <https://www.percona.com/doc/percona-server/5.7/flexibility/max_binlog_files.html#max_binlog_files>`_ system variable
 
+Deprecated Storage engine
+-----------------------------------------------------
+
+- TokuDB is deprecated in the 8.0 series and, after version 8.0.25, is not supported. Percona will remove this storage engine in the next major release of Percona Server for MySQL. We recommend MyRocks as a long-term migration path.
```
Comment (LenzGr) [inline doc/source/changed_in_version.rst]: The exact wording and time line is still under debate. Please withhold from merging this PR for now.

**Review by LenzGr — CHANGES_REQUESTED** [review]

File: doc/source/changed_in_version.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/changed_in_version.rst]: Updated the notes on other pages to match the RN. Added migration steps to Removing TokuDB.

**Review by george-lorch — APPROVED** [review]

**Review by LenzGr — APPROVED** [review]
LGTM! Thank you.

File: doc/source/tokudb/removing_tokudb.rst [hunk 2]
```diff
@@ -4,21 +4,46 @@
...
+
+To migrate data use the `mysqldump <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html>`__ client utility or the tools in the `MySQL Workbench <https://dev.mysql.com/downloads/workbench/>`__ to dump and restore the database.
+
+We recommended migrating to the `MyRocks` storage engine. Follow these steps to migrate the data:
+
+1. Use mysqldump to backup the TokuDB database into a single file.
+
+2. Create a MyRocks instance with MyRocks tables with no data.
+
+3. Remove the references to `TokuDB` and replace with `MyRocks`.
```
Comment (inikep) [inline doc/source/tokudb/removing_tokudb.rst]:
```suggestion
3. Replace the references to `TokuDB` with `MyRocks`.
```

Comment (patrickbirch) [reply doc/source/tokudb/removing_tokudb.rst]: Done.

**Review by inikep — APPROVED** [review]
LGTM

**Pull request merged**